### ISSUE TITLE
v6.0.0 breaks v1beta11 using .env variables

### ISSUE THREAD

**Issue by madhavajay** [loft-sh/devspace#2241]
**What happened?**  
Our CI upgraded to v6.0.0 and our tests broke.
It seems like the environment variables we have in our .env are not being added to the old devspace file with v6.0.0 like they were before automatically with v5.18.5.
https://github.com/OpenMined/PySyft/blob/dev/packages/grid/devspace.yaml

I understand this might have been implicit and not defined but this causes the new cli tool to fail to load it. I manually added the values but the generated v1beta11 file with `devspace print` fails with this error:
```
build:vpn-iptables error: denied: requested access to the resource is denied
build_images: build images: error building image openmined/grid-vpn-iptables:ncgyvV: error executing 'docker buildx build --tag openmined/grid-vpn-iptables:ncgyvV --tag openmined/grid-vpn-iptables:latest --tag openmined/grid-vpn-iptables:0.7.0-beta.42 --push --file iptables.dockerfile -':
```

I will revert back for now as its possible were doing something weird that isn't compatible with the new format or cli tool.

**What did you expect to happen instead?**  
I should have pinned our devspace version so thats our bad.

**How can we reproduce the bug?** (as minimally and precisely as possible)  
Check our CI logs or clone our repo on `dev` and run `tox -e stack.test.integration.k8s` with devspace v6.0.0 installed.

My devspace.yaml:
See: https://github.com/OpenMined/PySyft/blob/dev/packages/grid/devspace.yaml

**Local Environment:**  
- DevSpace Version: [use `devspace --version`]
v6.0.0
- Operating System: windows | linux | mac
mac and linux
- ARCH of the OS: AMD64 | ARM64 | i386
x86
**Kubernetes Cluster:**  
k3d and docker
- Cloud Provider: google | aws | azure | other
- Kubernetes Version: [use `kubectl version`]

**Anything else we need to know?**  
devspace is awesome 😎

/kind bug

**Comment by Orgil** [comment]
i also had a same problem. So you should put below on devspace.yaml which loads all .env variables and passed to containers as env vars. But keep in mind that all env not replaced if you put env config on the container.
```yaml
vars:
  DEVSPACE_ENV_FILE: ".env"
```

**Comment by madhavajay** [comment]
@Orgil i tried that and it doesnt work, probably because the old syntax for `v1beta11` is:

```
vars:
  - name: CONTAINER_REGISTRY
    value: ""
  - name: DOCKER_IMAGE_DB
...
 ```

**Comment by Orgil** [comment]
@madhavajay it might be. I'm using v2beta1.

**Comment by FabianKramm** [comment]
@madhavajay thanks for reporting this issue! We'll fix that with the new v6.0.1 version!
